github-actions/scan/__test__/configuration-model-customizer.test.ts

```diff
@@ -0,0 +1,191 @@
+// SPDX-License-Identifier: MIT
+
+import { addAdditonalExcludes } from "../src/configuration-model-customizer";
+
+
+describe('addAdditonalExcludes', function () {
+    const EXPECTED_ADDITIONAL_EXCLUDES: string[] = ['**/.sechub-gha/**'];
+
+    it('data section entries updated correctly', function () {
+        /* prepare */
+        let sampleJson = {
+            "apiVersion": "1.0",
+            "data": {
+              "sources": [
+                {
+                  "excludes": [ 
+                    "**/mytestcode/**",
+                    "*.config"
+                  ]
+                },
+                {
+                   "name": "gamechanger-sources"
+                }
+              ],
+              "binaries": [
+                {
+                  "excludes": [ 
+                    "**/test/**"
+                  ]
+                },
+                {
+                  "name": "gamechanger-binaries"
+                }
+              ]
+            }
+          };
+
+        /* execute */
+        addAdditonalExcludes(sampleJson);
+
+        /* test */
+        sampleJson.data.sources.forEach((entry: { excludes?: string[]; }) => {
```

Comment:
Please add to following checks:
- sources.excludes must also still contain `"**/mytestcode/**", "*.config"` (to check we do not overwrite custom settings)
- binaries.exludes must also still contain `"**/test/**"`